lib/html-compiler.js

```diff
@@ -4,7 +4,7 @@ const fs = require('fs');
 const path = require('path');
 const UglifyJs = require('uglify-js');
 const urlJoin = require('url-join');
-const pify = require('pify');
+const p = require('util.promisify');
```

Comment:
I'm not really into the one-letter variable name. If you don't want to use `pify`, I suggest we use the `promisify` function name.